Source/Extensions/UIImageView+Imaginary.swift

```diff
@@ -1,5 +1,10 @@
 import UIKit
 
+public struct Imaginary {
```

Comment:
@zenangst I think it's better to have it here https://github.com/hyperoslo/Imaginary/blob/master/Source/Imaginary.swift